edgexfoundry/app-service-configurable
# feat: Enable Delay Start capability for ASC

closes #401

Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-service-configurable/blob/main/.github/CONTRIBUTING.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions
Run `make gen ds-virtual delayed-start asc-http app-dev` from compose builder
Run `docker-compose -p edgex up -d`
Verify `edgex-app-http-export` logs show it was able to obtain Vault token and connect to the secure EdgeX MessageBus

## New Dependency Instructions (If applicable)
